connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResourceTest.java

```diff
@@ -143,7 +152,7 @@ public void testListConnectors() throws Throwable {
 
         PowerMock.replayAll();
 
-        Collection<String> connectors = connectorsResource.listConnectors(FORWARD);
+        Collection<String> connectors = (Collection<String>)connectorsResource.listConnectors(false, forward);
```

Comment:
Would be great to add another unit test for listing connectors with expanded results, to serve as a regression test for future changes.